tests/qos/qos_sai_base.py

```diff
@@ -1670,6 +1671,9 @@ def dut_disable_ipv6(self, duthosts, get_src_dst_asic_and_duts, tbinfo, lower_to
 
         for duthost in get_src_dst_asic_and_duts['all_duts']:
             duthost.shell("sysctl -w net.ipv6.conf.all.disable_ipv6=0")
+            logger.info("Adding docker0's IPv6 address since it was removed when disabing IPv6")
+            duthost.shell("ip -6 addr add fd00::1/80 dev docker0")
```

Comment:
Though I find the address : fd00::1/80 is the default ipv6 address used during dockerd start, I feel it is good to fetch/save the ipv6 address before line 1688 above in a variable and use the same while restoring it back?

This will avoid any change to testcase in future if this ipv6 address is changed in docker config.

Reply:
Fetched and saved docker0's Ipv6 address and used it after enabling IPv6